test/unittests/ngsi10/UpdateContextRequest_test.cpp

```diff
@@ -100,7 +100,7 @@ TEST(UpdateContextRequest, badIsPattern_json)
 
    std::string out = jsonTreat(testBuf, &ci, &parseData, UpdateContext, "updateContextRequest", &reqP);
    EXPECT_STREQ(expectedBuf, out.c_str());
-   reqP->release(&parseData);
+//   reqP->release(&parseData);
```

Comment:
This is not a present() method... should the release() line be removed? @kzangeli what do you think?

Reply:
Good catch!
I would definitely not remove it ...
Would probably cause a memleak.

Reply:
fixed in 96e667a